DP02_11_User_Packages/DP02_11_Working_with_user_packages.ipynb

```diff
@@ -177,7 +177,7 @@
     "\n",
```

Comment:
<p>The "if you wish to use it" could be confusing to users who don't know enough to wish one way or the other. Easier to write the instructions to include the creation of the category. Note also above "WORKDIR" is used, and below it is "WORK". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I am a little bit conflicted here.  I understand the need to avoid confusing users, but I also don't want to insist that users create the same directory structure that I use (which in this case was, admittedly, a bit of a random choice on my part :-) ).  Maybe I could write the instructions for creating ~/WORK/GitHub, but in a following parenthetical comment note that it is not necessary to use this particular directory structure, especially if the user already has a directory for this purpose?

Good catch on WORK vs. WORKDIR!  I completely missed that!

Thanks!

Reply:
That sounds like a great compromise Douglas! Thanks